endpoints/openrtb2/video_auction.go

```diff
@@ -480,22 +522,28 @@ func createBidExtension(videoRequest *openrtb_ext.BidRequestVideo) ([]byte, erro
 	return reqJSON, nil
 }
 
-func (deps *endpointDeps) parseVideoRequest(request []byte) (req *openrtb_ext.BidRequestVideo, errs []error) {
+func (deps *endpointDeps) parseVideoRequest(request []byte) (req *openrtb_ext.BidRequestVideo, errs []error, podErrors []PodError) {
 	req = &openrtb_ext.BidRequestVideo{}
 
 	if err := json.Unmarshal(request, &req); err != nil {
 		errs = []error{err}
 		return
 	}
 
-	errL := deps.validateVideoRequest(req)
+	errL, podErrors := deps.validateVideoRequest(req)
 	if len(errL) > 0 {
 		errs = append(errs, errL...)
 	}
 	return
 }
 
-func (deps *endpointDeps) validateVideoRequest(req *openrtb_ext.BidRequestVideo) []error {
+type PodError struct {
+	PodId     int
+	PodIndex  int
+	PodErrors []string
```

Comment:
Can we please call this either `errMsgs` or something along those lines? It's just that `PodError.PodErrors` can become real difficult to follow real soon when reading the code

Reply:
Yes, done.